src/idl_gen_csharp.cpp

```diff
@@ -1990,6 +1990,19 @@ class CSharpGenerator : public BaseGenerator {
           "Newtonsoft.Json.Formatting.Indented);\n";
       code += "  }\n";
     }
+    if (parser_.root_struct_def_ == &struct_def) {
+      code += "  public static " + class_name +
+              " DeserializeFromFlatBuffers(byte[] fbBuffer) {\n";
+      code += "    return " + struct_def.name + ".GetRootAs" + struct_def.name +
+              "(new ByteBuffer(fbBuffer)).UnPack();\n";
+      code += "  }\n";
+      code += "  public byte[] SerializeToFlatBuffers() {\n";
```

Comment:
SerializeToBinary? Given that this function is part of FlatBuffers, the name might be superfluous?

Reply:
renamed to  `SerializeToBinary`.